libraries/AP_HAL_Linux/Perf.cpp

```diff
@@ -90,59 +108,56 @@ Util::perf_counter_t Util::perf_alloc(perf_counter_type type, const char *name)
     }
     }
 
-    base->name = name;
-    base->type = type;
-    return (perf_counter_t)base;
-}
+#ifdef HAVE_LIBSYSTEMD
+    PerfManager::getInstance()->add((Util::perf_counter_t)perf);
+#endif
 
-static inline uint64_t timespec_to_nsec(const struct timespec *ts)
-{
-    return ts->tv_nsec + (ts->tv_sec * NSEC_PER_SEC);
+    return (Util::perf_counter_t)perf;
```

Comment:
@lucasdemarchi What I was referring regarding _Util::_ was this, you are inside the Util class so why is the _Util::_ part necessary?